function/ImageFunctions.cs

```diff
@@ -0,0 +1,144 @@
+using System;
+using System.IO;
+using System.Threading.Tasks;
+using System.Text.Json;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Azure.WebJobs;
+using Microsoft.Azure.WebJobs.Extensions.Http;
+using Microsoft.AspNetCore.Http;
+using Microsoft.Extensions.Logging;
+using Microsoft.WindowsAzure.Storage.Blob;
+using Surfrider.PlasticOrigins.Backend.Mobile.Service.Auth;
+using Surfrider.PlasticOrigins.Backend.Mobile.Service;
+using Surfrider.PlasticOrigins.Backend.Mobile.ViewModel;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace Surfrider.PlasticOrigins.Backend.Mobile
+{
+    public class ImageFunctions
+    {
+        private IImageService _imageService;
+
+        public ImageFunctions(IImageService imageService)
+        {
+            _imageService = imageService;
+        }
+
+        [FunctionName("GetOneImage")]
+        public string RunGetOneImage(
+            [HttpTrigger(AuthorizationLevel.Anonymous, "get", Route = "images/imgName/{fileName}")] HttpRequest req,
+            [Blob("testforlabelbackend", FileAccess.Read, Connection = "TraceStorage")] CloudBlobContainer blobContainer,
+            [AccessToken] AccessTokenResult accessTokenResult,
+            string fileName,
+            ILogger log
+        )
+        {
+            log.LogInformation($"Get Image {fileName}");
+
+            if (accessTokenResult.Status != AccessTokenStatus.Valid)
```

Comment:
@AntoineBruge La partie labelling doit rester ouverte à tous, sans compte ?

Reply:
Yes c'est exacte!
A user can label images without login in.
A user can upload images but he has to login in first.

Reply:
@benzinamohamedelyes pour info!